docker-compose.yml

```diff
@@ -1,7 +1,7 @@
-version: '3.8'
+
 services:
   ovirt-postgres:
-    image: postgres:13.14
+    image: postgres:13.16
```

Comment:
ovirt-engine just depends on postgresql-server >= 12.0
This means that everytime a new postgresql package is uploaded onto centos, we would need to bump this.
Eventually change it to postgres:13 for now until we have a better solution?

Reply:
i've updated to use postgres:13 without the minor version.